platform/features/layout/src/FixedController.js

```diff
@@ -21,8 +21,22 @@
  *****************************************************************************/
 
 define(
-    ['./FixedProxy', './elements/ElementProxies', './FixedDragHandle'],
-    function (FixedProxy, ElementProxies, FixedDragHandle) {
+    [
+        'lodash',
+        './FixedProxy',
+        './elements/ElementProxies',
+        './FixedDragHandle',
+        '../../table/src/TelemetryCollection',
+        '../../../../src/api/objects/object-utils'
```

Comment:
Would be nice to have the object-utils available as shortcuts, perhaps on the API.  There's a number of cases where you want a string identifier and it should be easier to use them.

I think this is not the correct use case for telemetry collection.  fixed position shows one value-- why does it track many?